pr-deploy/src/github.ts

```diff
@@ -19,13 +19,19 @@ import {ChecksListForRefParams, ChecksUpdateParams} from '@octokit/rest';
 import {GitHubAPI} from 'probot/lib/github';
 
 export class PullRequest {
+  public headSha: string;
+  private action: Octokit.ChecksUpdateParamsActions;
   private check: string;
   private github: GitHubAPI;
-  public headSha: string;
   private owner: string;
   private repo: string;
 
   constructor(github: GitHubAPI, headSha: string, owner: string, repo: string) {
+    this.action = {
```

Comment:
Since this remains the same across all instances of `PullRequest`, you can make it a module-level const